bigtop-packages/src/common/hadoop/install_hadoop.sh

```diff
@@ -177,8 +177,28 @@ export HADOOP_LIBEXEC_DIR=/${SYSTEM_LIBEXEC_DIR#${PREFIX}}
 exec ${component#${PREFIX}} "\$@"
 EOF
   chmod 755 $wrapper
+  
+  # ODPI-131 Add wrappers for hdfs, yarn, mapred in $HADOOP_DIR
```

Comment:
Lines 170-178 and 186-194 are identical, so the generation code for the wrapper script should probably be put in a separate function.